packages/dx-react-grid-demos/package.json

```diff
@@ -44,6 +44,7 @@
     "codemirror": "^5.49.2",
     "core-js": "^3.4.1",
     "d3-shape": "^1.3.7",
+    "faker": "^4.1.0",
```

Comment:
It may be a problem😢

Reply:
320kB, I see. Agreed

Reply:
No, a problem related to publishing a custom data on our site. I know, it sounds like a joke, but it's true. I propose using data sources from js.devexpress.com.

Reply:
Then I'll just use the json from the form edit demo of dxgrid

Reply:
Ok, it would be better